qtfred/src/mission/missionsave.cpp

```diff
@@ -3701,10 +3701,10 @@ int CFred_mission_save::save_variables() {
 				}
 
 				// player-persistent - Goober5000
-				if (Sexp_variables[i].type & SEXP_VARIABLE_PLAYER_PERSISTENT) {
+				if (Sexp_variables[i].type & SEXP_VARIABLE_SAVE_ON_MISSION_CLOSE) {
 					fout("\t\t\"%s\"", "player-persistent");
```

Comment:
This should be changed to `save-on-mission-close` to match the non-qtFRED version of missionsave.cpp.